test/test.js

```diff
@@ -650,6 +651,85 @@ const createTestSuite = (useElectronNet) => {
       })
     })
 
+    it('should handle aborts before request', function () {
+      this.timeout(500)
```

Comment:
This timeout doesn't seem useful in this case

Reply:
Got rid of spurious timeouts